### ISSUE TITLE
[a11y] Cannot change other user's username

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1505]
![msedge_CpcUGM0Ns8](https://user-images.githubusercontent.com/37256050/167700666-ebc25ad1-48ff-47e5-a074-84b752892e12.gif)

Related to issue https://github.com/specify/specify7/issues/1504

db: colorado_paleo
username: topdog_CUpaleo

**Comment by maxxxxxdlp** [comment]
Duplicate of #1480
